src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
```

Comment:
Avoid using generic variable names like 'styles'. It decreases code readability. Consider using a more descriptive name.